src/validation.c

```diff
@@ -72,6 +72,11 @@ validate_tunnel_grammar(NetplanNetDefinition* nd, yaml_node_t* node, GError** er
     if (!nd->tunnel.remote_ip)
         return yaml_error(node, error, "%s: missing 'remote' property for tunnel", nd->id);
 
+    if (nd->tunnel.ttl) {
+	if ((nd->tunnel.ttl) < 1 || (nd->tunnel.ttl) > 255)
```

Comment:
I think we can combine the two `if`s.. if `(nd->tunnel.ttl)` is true it will be > 0 (as this is a uint variable), which makes the check for `nd->tunnel.ttl < 1` useles.

So we could just combine it as: `if (nd->tunnel.ttl && nd->tunnel.ttl > 255)`